src/Dapr.Jobs/Models/DaprJobSchedule.cs

```diff
@@ -0,0 +1,146 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2024 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using Dapr.Jobs.Extensions;
+using Dapr.Jobs.JsonConverters;
+
+namespace Dapr.Jobs.Models;
+
+/// <summary>
+/// Used to build a schedule for a job.
+/// </summary>
+[JsonConverter(typeof(DaprJobScheduleConverter))]
+public sealed class DaprJobSchedule
+{
+    /// <summary>
+    /// A regular expression used to evaluate whether a given prefix period embodies an @every statement.
+    /// </summary>
+    private readonly Regex isEveryExpression = new(@"^@every (\d+(m?s|m|h))+$", RegexOptions.Compiled);
```

Comment:
`static`?  (Here and next property below.)

Reply:
Done